tests/manual/test_memory_reporting.py

```
@@ -0,0 +1,293 @@
+    for _ in range(timeout):
+        info = get_collection_info(collection_name)
+        status = info.get("status")
+        if status == "green":
+            return
+        time.sleep(1)
+    raise TimeoutError(f"Collection {collection_name} did not reach green status in {timeout}s")
+
+
+def delete_payload_index(collection_name: str, field_name: str):
+    """Delete a payload index."""
+    resp = requests.delete(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)